pkg/identityserver/bunstore/api_key_store.go

```diff
@@ -359,7 +359,7 @@ func (s *apiKeyStore) DeleteEntityAPIKeys(ctx context.Context, entityID *ttnpb.E
 	))
 	defer span.End()
 
-	entityType, entityUUID, err := s.getEntity(ctx, entityID)
+	entityType, entityUUID, err := s.getEntity(store.WithSoftDeleted(ctx, false), entityID)
```

Comment:
Was this possible with `gorm`, but not `bun` ?

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/blob/ac05bfe303c360973eae155d3b827bfa7daa8f50/pkg/identityserver/gormstore/api_key_store.go#L221-L226

But I did not check for everyone, just changed in a way that local test would work. Going to double check it